.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
```

Comment:
This is needed to instruct various text editors how to handle the line indentation, and in fibernavigator's case, line end - since it's dos formatted. You are right, it should be detected by most text editor, but many don't, or do it incorrectly. I had to add this for vim, for example. Most major text editor supports .editorconfig, and if not, it's simply ignored obviously.

This page has a list of text editors that supports editorconfig
> http://editorconfig.org/